Automattic/wp-calypso
# Use new domains illustrations

#### Changes proposed in this Pull Request

We have some new illustrations to use for the domains pages. This PR just replaces some of the existing illustrations with these new ones.

#### Testing instructions

For sites with a plan, but no domain, make sure that you see the new `free-domain.svg` in the domain list:

<img width="1085" alt="Screen Shot 2021-12-07 at 1 19 54 PM" src="https://user-images.githubusercontent.com/1379730/145090987-5dbf6b1c-00e6-4daf-8360-9e49575cee50.png">

<img width="1085" alt="Screen Shot 2021-12-07 at 1 20 57 PM" src="https://user-images.githubusercontent.com/1379730/145091004-fa182fa7-0c08-4b84-9653-8158a0964990.png">

<img width="370" alt="Screen Shot 2021-12-07 at 1 21 26 PM" src="https://user-images.githubusercontent.com/1379730/145091013-f5cf1911-bd08-4284-9229-3c66b6922941.png">

Go through the connect a domain flow and on the last page, make sure you see the new "connect" illustration:

<img width="1103" alt="Screen Shot 2021-12-07 at 1 23 59 PM" src="https://user-images.githubusercontent.com/1379730/145091191-011f5246-d276-4fd6-9b08-2056816251c8.png">

On the "Use a domain I own" flow, make sure that the input page shows the new "domain" illustration:

<img width="1080" alt="Screen Shot 2021-12-07 at 1 38 36 PM" src="https://user-images.githubusercontent.com/1379730/145091418-e8093e2f-7368-4ec3-b70f-c87a3a8b9c5b.png">

...and that the next page shows the new transfer and new connection illustrations:

<img width="1078" alt="Screen Shot 2021-12-07 at 12 56 41 PM" src="https://user-images.githubusercontent.com/1379730/145091536-641a7622-c0b7-422e-a45c-b9d53a9b3fbd.png">

Checkout pages:

<img width="1163" alt="Screen Shot 2021-12-07 at 2 19 14 PM" src="https://user-images.githubusercontent.com/1379730/145092635-d3e8f585-7806-4edd-a7ab-2b468ac832c4.png">

<img width="1163" alt="Screen Shot 2021-12-07 at 2 20 30 PM" src="https://user-images.githubusercontent.com/1379730/145092660-e2134d18-501f-4e88-8cbd-a0fdf754e57b.png">

<img width="1163" alt="Screen Shot 2021-12-07 at 2 22 48 PM" src="https://user-images.githubusercontent.com/1379730/145092920-bfd95c0f-3f38-4238-9224-77eaa61cc6de.png">
